gui/src/containers/Bridges.js

```diff
@@ -44,7 +44,7 @@ export class Bridges extends Component {
               Bridges
             </Typography>
           </Grid>
-          <Button variant='outlined' color='primary' component={Link} to={'/create/bridge'}>
+          <Button variant='outlined' color='primary' component={ReactStaticLinkComponent} to={'/create/bridge'}>
```

Comment:
There is now a `LinkButton` component. How do you feel about making it more generic so that it can be used here?

https://github.com/smartcontractkit/chainlink/blob/master/gui/src/components/LinkButton.js

Reply:
I don't think there's anything I can do with the `LinkButton` component, because it is a button and acts like a button, whereas `ReactStaticLinkComponent` is a react-static `Link` that inherits all the button styles to be more beautiful.